examples/service-catalog/service-catalog.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
+- kind: ClusterRoleBinding
```

Comment:
right now it may just require create, i am sure in the future it will require update - if it doesn't already. I am not sure whether it is handling delete itself or relying on GC for that, @pmorie ?